packages/expect/src/__tests__/matchers-toContain.property.test.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import fc from 'fast-check';
+import {
+  anythingSettings,
+  assertSettings,
+} from './__arbitraries__/sharedSettings';
+
+describe('toContain', () => {
+  it('should always find the value when inside the array', () => {
+    fc.assert(
+      fc.property(
+        fc.array(fc.anything(anythingSettings)),
+        fc.array(fc.anything(anythingSettings)),
+        fc.anything(anythingSettings).filter(v => !Number.isNaN(v)),
+        (startValues, endValues, v) => {
+          // Given:  startValues, endValues arrays and v value (not NaN)
+          // Assert: We expect `expect([...startValues, v, ...endValues]).toContain(v)`

```

Comment:
can we get rid of this comment (and alike)? below line is a duplicate of it :)

Reply:
Do you want me to remove all the comments? Or only some of them?

Reply:
Just the comments that result are duplicates of the line below (so any "// Assert: ..." will likely match)

Reply:
Just fixed it with my last commit